indico/MaKaC/webinterface/tpls/RoomBookingSearchRoomsResults.tpl

```diff
@@ -90,3 +90,22 @@
   </tr>
 </table>
 <br />
+<script type="text/javascript">
+  $(document).ready(function() {
+      $.fn.extend({
```

Comment:
There are some problems when I don't do it that way. Maybe I will spend a bit more time to investigate it.